src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file LoginForm.js
+ * Exports a component that allows users to log into EditVR.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { TextField, Button, withStyles } from '@material-ui/core';
+
+import LoginFormStyles from './LoginForm.style';
+
+class LoginForm extends Component {
+  static propTypes = {
+    classes: PropTypes.shape({
+      textField: PropTypes.string.isRequired,
+      button: PropTypes.string.isRequired
+    }).isRequired
+  };
+
+  state = {
```

Comment:
If this state is driving some other interaction (validation, animation) other than the text box, this is a good use of state. otherwise, why not just get the target value when the user submits the form? Like the good `ole days.